src/main/java/com/minecolonies/structures/helpers/Structure.java

```diff
@@ -155,7 +155,7 @@ public boolean isTemplateNull()
      */
     public boolean doesExist()
     {
-        return template == null ? false : true;
+        return !isTemplateNull();
```

Comment:
I wondered the same thing, but I just used it because it was there, and `template == null ? false : true` was really bad. I'll look into it.